modules/update-preparing-list.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-prepare.adoc
+
+[id="update-preparing-list_{context}"]
+= Removed Kubernetes APIs
+
+// TODO: Keep michael's section in the release notes (which this duplicates), or link to this from his RN section?
+{product-title} 4.9 uses Kubernetes 1.22, which removed the following deprecated `v1beta1` APIs. Migrate manifests and API clients to use the `v1` API version. For more information about migrating removed APIs, see the link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-22[Kubernetes documentation].
```

Comment:
Consider starting the second sentence as, "You must migrate manifests..." or even "You must migrate API clients and manifests..." to avoid the alliteration. 😁